src/components/calcite-inline-editable/calcite-inline-editable.e2e.ts

```diff
@@ -0,0 +1,509 @@
+    const element = await page.find("calcite-input");
+    const clearButton = await page.find(".calcite-input-clear-button");
+    expect(element.getAttribute("value")).toBe("John Doe");
+    expect(calciteInputInput).toHaveReceivedEventTimes(0);
+    clearButton.click();
+    await page.waitForChanges();
+    expect(element.getAttribute("value")).toBe("");
+    expect(calciteInputInput).toHaveReceivedEventTimes(1);
+  });
+
+  it("when type is search and input is cleared via escape key, event is received", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-input type="search" value="John Doe"></calcite-input>
+    `);
+
+    const calciteInputInput = await page.spyOnEvent("calciteInputInput");
+    const element = await page.find("calcite-input");
+    expect(element.getAttribute("value")).toBe("John Doe");
+    await element.callMethod("setFocus");
+    expect(calciteInputInput).toHaveReceivedEventTimes(0);
+    await page.keyboard.press("Escape");
+    await page.waitForChanges();
+    expect(element.getAttribute("value")).toBe("");
+    expect(calciteInputInput).toHaveReceivedEventTimes(1);
+  });
+
+  it("when clearable is not requested and input is cleared via escape key, event is not received", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-input value="John Doe"></calcite-input>
+    `);
+
+    const calciteInputInput = await page.spyOnEvent("calciteInputInput");
+    const element = await page.find("calcite-input");
+    expect(element.getAttribute("value")).toBe("John Doe");
+    await element.callMethod("setFocus");
+    expect(calciteInputInput).not.toHaveReceivedEvent();
+    await page.keyboard.press("Escape");
+    await page.waitForChanges();
+    expect(element.getAttribute("value")).toBe("John Doe");
+    expect(calciteInputInput).not.toHaveReceivedEvent();
+  });
+
+  it("should emit event when up or down clicked on input", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-input type="number" max="0" value="-2"></calcite-input>
+    `);
+
+    const calciteInputInput = await page.spyOnEvent("calciteInputInput");
+
+    const numberHorizontalItemUp = await page.find(
+      "calcite-input .calcite-input-number-button-item[data-adjustment='up']"
+    );
+    expect(calciteInputInput).toHaveReceivedEventTimes(0);
+    await numberHorizontalItemUp.click();
+    await page.waitForChanges();
+
+    expect(calciteInputInput).toHaveReceivedEventTimes(1);
+
+    const numberHorizontalItemDown = await page.find(
+      "calcite-input .calcite-input-number-button-item[data-adjustment='down']"
+    );
+    await numberHorizontalItemDown.click();
+    await page.waitForChanges();
+    expect(calciteInputInput).toHaveReceivedEventTimes(2);
+
+    await numberHorizontalItemDown.click();
+    await page.waitForChanges();
+    expect(calciteInputInput).toHaveReceivedEventTimes(3);
+  });
+});
```

Comment:
Will there be any tests for `calcite-inline-editable`?   Looks like this file was copy pasta'd from `calcite-input.e2e.ts` ☝️